### PR TITLE
feat(IR-1829-left-navbar): improvements on Accordion Tailwind component

### PR DESCRIPTION
**Pull request by kattsushi**
## Summary
feat(IR-1829-left-navbar): improvements on Accordion Tailwind component
## References
this one blocks [#211](https://github.com/theinfinitereality/eepro-multitenancy/pull/211)
closes #_insert number here_

### CODE REVIEW

**Review by hanzlamateen — APPROVED** [review]
Looks good to me, though @aditya-mitra can give it a second look.

File: packages/ui/src/primitives/tailwind/Accordion/index.tsx [hunk 1]
```diff
@@ -58,14 +72,18 @@ const Accordion = forwardRef(
             openState.set((v) => !v)
           }}
         >
-          <Text component="h2" fontSize="xl" fontWeight="semibold">
-            {title}
-          </Text>
+          <div className={twClassNameTitle}>
+            {prefixIcon && <div className="mr-2">{prefixIcon}</div>}
+            <Text component="h2" fontSize="xl" fontWeight="semibold">
+              {' '}
```
Comment (aditya-mitra) [inline packages/ui/src/primitives/tailwind/Accordion/index.tsx]: we should remove this space

File: packages/ui/src/primitives/tailwind/Accordion/index.tsx [hunk 2]
```diff
@@ -33,17 +33,31 @@ export interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
...
-    { title, subtitle, expandIcon, shrinkIcon, children, className, open, ...props }: AccordionProps,
+    {
+      title,
+      subtitle,
+      expandIcon,
+      shrinkIcon,
+      prefixIcon,
+      children,
+      className,
+      classNameTitle,
```
Comment (aditya-mitra) [inline packages/ui/src/primitives/tailwind/Accordion/index.tsx]: we should use `titleClassName` instead of `classNameTitle`

**Review by aditya-mitra — CHANGES_REQUESTED** [review]

**Review by aditya-mitra — APPROVED** [review]
LGTM

**Review by hanzlamateen — APPROVED** [review]

**Comment by HexaField** [comment]
@hanzlamateen this PR introduced type errors

PR fix here https://github.com/EtherealEngine/etherealengine/pull/10152

**Pull request merged**